azure-tools-common/src/main/java/com/microsoft/azure/common/applicationinsights/ApplicationInsightsManager.java

```diff
@@ -13,7 +13,7 @@
 import com.microsoft.azure.management.applicationinsights.v2015_05_01.ApplicationType;
 import com.microsoft.azure.management.applicationinsights.v2015_05_01.implementation.InsightsManager;
 import com.microsoft.azure.management.resources.Provider;
-import com.microsoft.azure.tools.common.util.ProxyUtils;
+import com.microsoft.azure.toolkit.lib.common.proxy.ProxyManager;
```

Comment:
remove, the manager doesn't need to know proxy

Reply:
fixed